src/applications/facility-locator/components/ProviderServiceDescription.jsx

```diff
@@ -8,9 +8,9 @@ const providerName = (query, posCodes) => {
       name = facilityTypes.cc_pharmacy.toUpperCase();
       break;
     case 'urgent_care':
-      if (posCodes && posCodes === 17) {
+      if (posCodes && parseInt(posCodes, 0) === 17) {
         name = ccUrgentCareLabels.WalkIn;
-      } else if (posCodes && posCodes === 20) {
+      } else if (posCodes && parseInt(posCodes, 0) === 20) {
```

Comment:
What does a parameter of `0` mean here? Did you mean `10`?

Reply:
yes, should 10. thanks for the catch